src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -0,0 +1,1150 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This file looks similar to:
https://github.com/dotnet/corefx/blob/master/src/System.Data.Common/src/System/Data/Common/AdapterUtil.cs
Can they be merged rather than duplicated?